cse442-at-ub/project_s23-webarebears
# Search bar backend functionality #77 (Duplicate)

**Task Test**

*Test 1*

1) Go to https://www-student.cse.buffalo.edu/CSE442-542/2023-Spring/cse-442ac/backend/login.php
2) Login using username: abc123 password:abc123
3) From the application, navigate to any page and use the search bar to search up a username. Account that exist: Junstin
4) If the username exist, it should be display below the search bar and you should be able to add them as friend.
5) If the username does not exist, it should display a message saying "No results found."
